sources/geom.json

```diff
@@ -3,8 +3,8 @@
 	"fullname": "geom",
 	"name": "geom",
 	"prefix": "geom",
-    	"key":"id",
+	"key": "id",
```

Comment:
Keeping `addr.json`, `edtf.json`, `geom.json`, and `name.json` in the repo. 

Only adding `"Property prefix."` in the `"description"`.